serviceorchestration-simple/src/main/java/eu/arrowhead/serviceorchestration/service/normalization/OrchestrationHistoryManagementNormalization.java

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2025 AITIA
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ *
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  	AITIA - implementation
+ *  	Arrowhead Consortia - conceptualization
+ *
+ *******************************************************************************/
+
+package eu.arrowhead.serviceorchestration.service.normalization;
+
+import eu.arrowhead.common.Utilities;
+import eu.arrowhead.common.service.PageService;
+import eu.arrowhead.common.service.validation.name.ServiceDefinitionNameNormalizer;
+import eu.arrowhead.common.service.validation.name.SystemNameNormalizer;
+import eu.arrowhead.dto.OrchestrationHistoryQueryRequestDTO;
+import eu.arrowhead.dto.enums.OrchestrationType;
+import eu.arrowhead.serviceorchestration.jpa.entity.OrchestrationJob;
+import eu.arrowhead.serviceorchestration.service.enums.OrchestrationJobStatus;
+import eu.arrowhead.serviceorchestration.service.model.NormalizedOrchestrationJobQueryRequest;
+import eu.arrowhead.serviceorchestration.service.normalization.utils.OrchestrationNormalization;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+
+@Service
+public class OrchestrationHistoryManagementNormalization {
+
+	//=================================================================================================
+	// members
+
+	@Autowired
+	private SystemNameNormalizer systemNameNormalizer;
+
+	@Autowired
+	private OrchestrationNormalization orchNormalizer;
+
+	@Autowired
+	private ServiceDefinitionNameNormalizer serviceDefNameNormalizer;
+
+	@Autowired
+	private PageService pageService;
+
+	private final Logger logger = LogManager.getLogger(this.getClass());
+
+	//=================================================================================================
+	// methods
+
+	//-------------------------------------------------------------------------------------------------
+	public NormalizedOrchestrationJobQueryRequest normalizeOrchestrationHistoryQueryRequestDTO(final OrchestrationHistoryQueryRequestDTO dto, final String origin) {
+		logger.debug("normalizeOrchestrationHistoryQueryRequestDTO started...");
```

Comment:
I think in this method all new ArrayList<>() expressions can be replaced with List.of() expression, since these lists can be inmutables.